9.1_Community_Design.md

```diff
@@ -2,45 +2,33 @@
 
 ### 9.1.1 Defining Resources
 
-Communities can have a type of **ability** called a **resource** that your GM defines. Your PC can try to draw on their community's **resources** to use them as **abilities**. Your GM should focus on no more than five or so broadly-labeled **resource** types, so that the PCs can care about (and have a chance of successfully managing) all of them.
+Communities **resource** that your GM defines. Your PC can try to draw on their community's **resources** to use them as **bonus**. If your community is in difficulty, a strained **resource** might act as a **penalty**. Your GM should focus on no more than five or so broadly-labeled **resource** types, so that the PCs can care about (and have a chance of successfully managing) all of them.
```

Comment:
Communities have (missing word) resources that your GM defines.

Reply:
Thanks, fixed